src/django/pfb_analysis/views.py

```diff
@@ -355,3 +360,29 @@ def get(self, request, format=None, *args, **kwargs):
                                            if sub['code'] in states_with_jobs]
 
         return Response(countries)
+
+
+class CrashesGeojsonViewSet(APIView):
+    """For retrieving all crash points as GeoJSON feature collection."""
+
+    filter_class = None
+    pagination_class = None
+    permission_classes = (RestrictedCreate, IsAuthenticatedOrReadOnly)
+    serializer_class = None
+
+    def get(self, request, format=None, *args, **kwargs):
+        analysis_job_uuid = request.GET.get('uuid')
+        analysis_job = get_object_or_404(AnalysisJob, pk=analysis_job_uuid)
+        neighborhood_geometry = analysis_job.neighborhood.geom
```

Comment:
```suggestion
        neighborhood_geometry = analysis_job.neighborhood.geog
```
I guess we could also rename the variable to e.g. `neighborhood_boundary` to avoid muddling the distinction.